docs/user_guide/sql-ai-shell.md

```diff
@@ -52,6 +52,16 @@ You can configure the AI Shell using the Beekeeper Studio [configuration system]
 
 {% ini-include section="plugins.bks-ai-shell" %}
 
+## Custom Instructions
+
+You can provide custom instructions to personalize how the AI Shell responds to your queries. To add custom instructions:
+
+1. Open the settings menu (⚙️).
+2. Go to the General tab.
+3. Add your custom instructions in the provided text field
+
+![AI Custom Instructions](../assets/images/ai/ai-custom-instructions.png)
```

Comment:
Questions this needs to answer:
1. It should tell the user it applies to ALL AI sessions, regardless of the connection or workspace
2. Does this get sent on every prompt, or just at the beginning of the session?

Reply:
Something to think about when providing settings:

Users are editing settings while they're connected to a specific database, so it would be natural for them to think that the settings only related to that database. That means we need to be explicit about when settings are applied globally - both in the app, and in the docs.

Also - this config system is implemented totally differently to the main configuration? Shouldn't this use the configuration files? I know we added a way for plugins to get/set credentials, but should this be how all configuration is stored?

I think we should provide these options as configuration options -- plugins should probably get their own configuration section in the config files?